sos/report/plugins/openvswitch.py

```diff
@@ -29,6 +29,11 @@ def setup(self):
             '/usr/local/var/log/openvswitch/',
         ]
 
+        dpdk_enabled = self.collect_cmd_output(
+            "ovs-vsctl -t 5 get Open_vSwitch . other_config:dpdk-init")
+        check_dpdk = (dpdk_enabled["status"] == 0 and
+                      dpdk_enabled["output"].startswith('"true"'))
```

Comment:
I know it's not used much (or at all?), but "try" is a valid value here too, in which case you would want to check the mempools. What is the advantage of the conditional? It may save a few cmds, but not sure if it's more trouble than it's worth.

Tested that with "true" the mempool information is present. Actually, it's called twice in my case due to dpdk ports being in port and interface lists, but I don't see that as an issue.

Reply:
I agree - I don't think anyone use it in production currently.  Actuallly, until DPDK has a robust uninit and reinit mechanism, "try" will not be recommended or used - I would prefer to wait until the uninit / reinit path is better.  For now "try" is just a debug tool for system integrators.

Reply:
ok, as "try" is _intended_ for use as a debug method when DPDK initialization is failing and in that case mempool information will not be available, the current approach seems fine.